src/lib/components/ContentfulRichText/nodes/Hr.svelte

```diff
@@ -1,14 +1 @@
-<script lang="ts">
-  import type { Node as NodeType, Hr } from "@contentful/rich-text-types";
-  import { isHr } from "../predicates";
-
-  export let node: NodeType;
-
-  let hr: Hr;
-  if (!isHr(node)) {
-    throw new Error("node is not an hr");
-  }
-  hr = node;
-</script>
-
```

Comment:
Was seeing some warnings popping up after this was removed; I think we can do this instead?

```
<script lang="ts">
  import type { Node as NodeType } from "@contentful/rich-text-types";
  import { isHr } from "../predicates";
  export let node: NodeType;
  if (!isHr(node)) {
    throw new Error("node is not an hr");
  }
</script>
```